site/content/media-kit.md

```diff
@@ -7,28 +7,37 @@ description: Download our assets and discover our style guide.
 
 ## Logos
 
-### Version 1
+### Version 1 - Rectangular
 
-![Logo - V1](https://i.imgur.com/ATxChJD.png)
+![Logo - ](https://i.imgur.com/VCqpxcT.png)
 
-### Version 2
+### Version 2 - Rectangular with no URL
 
-![Logo - V2](https://i.imgur.com/mS3GR9l.png)
+![Logo - V2](https://i.imgur.com/qny5XYN.png)
 
-### Version 3
+### Version 3 - Squared
+
+![Logo - V3](https://i.imgur.com/eFTD651.png)
+
+### Version 4 - Squared No URL
+
+![Logo - V4](https://i.imgur.com/RRVpP6V.png)
 
-![Logo - V3](https://i.imgur.com/mLO2a2t.png)
 
 ## Banners
 
 ### Banner 1
 
-![Banner 1](https://i.imgur.com/A43r74s.png)
+![Banner 1](https://i.imgur.com/QZAbJUJ.png)
 
 ### Banner 2
 
-![Banner 2](https://i.imgur.com/qulc21w.png)
+![Banner 2](https://i.imgur.com/EoAZnOp.jpg)
 
 ## Example Testimonial
 
-![Testimonial 1](https://i.imgur.com/6NkbPaD.png)
+![Testimonial 1](https://i.imgur.com/zptsdmw.jpg)
+
+## Imgur Album
+
+[Find all images here](https://imgur.com/a/xfBpacn)
```

Comment:
We need to add the banner image into the Imgur album as well.

Reply:
Forgot to login before creating that album, I'll create a new account and new album

Reply:
Imgur album is now at https://techqueria.imgur.com/ - I DMed you credentials